submissions/yaripey/oop-exercise/index.js

```diff
@@ -0,0 +1,103 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const male = 'male'
+const female = 'female'
+
+class Inhabitant {
+  constructor(species, name, legs, hands, gender, message) {
+    this.species = species
+    this.name = name
+    this.legs = legs
+    this.gender = gender
+    this.message = message
+    this.friendship = null
+    this.hands = hands
```

Comment:
Not every inhabitant has hands by natural design. The beauty of OOP is that every class has props it really needs.
QUick scan didn't show any hands count output